app/src/main/res/layout/activity_main.xml

```diff
@@ -13,7 +13,7 @@
         android:layout_marginStart="24dp"
         android:ems="10"
         android:inputType="textPersonName"
-        android:text="@string/edit_message"
```

Comment:
I am going to start by saying this is trivial with such a simple application, but I would say this on an application I currently develop on. I am not sure if you are aware of this but "@string/edit_message" is a variable pointing to the string variable file as you have removed the reference here, you should remove the variable. 
If you want help finding this, press ctrl + shift + f, then find "edit_message" you should find this variable and remove delete it.